AmplifyPlugins/DataStore/Tests/AWSDataStorePluginTests/Sync/RemoteSyncEngineTests.swift

```diff
@@ -46,16 +46,21 @@ class RemoteSyncEngineTests: XCTestCase {
     }
 
     func testErrorOnNilStorageAdapter() throws {
+        guard let remoteSyncEngine else {
```

Comment:
this may not work with Swift 5.6 which we still support. please check with Xcode 13.4.1.